data/README.md

```diff
@@ -0,0 +1,50 @@
+# meanbee/magento2-data
+
+Docker data images with Magento 2 source code and bi-directional file syncing.
+
+## Usage
+
+The images are intended to be used as [data containers](https://docs.docker.com/engine/tutorials/dockervolumes/#creating-and-mounting-a-data-volume-container), e.g.:
+
+    version: "2"
+    services:
+        data:
+            image: meanbee/magento2-data:2.1
+            privileged: true
```

Comment:
Good point. Not for the basic use case, but we do need it for Unison due to setting the inotify watches.